### PR TITLE
(RE-7431) Add pl-build-tools-staging repos

### PR DESCRIPTION
**Pull request by shrug**

### CODE REVIEW

**Comment by geoffnichols** [comment]
👍

File: builder_data.yaml [hunk 1]
```diff
@@ -9,21 +9,21 @@ svr4_host: pl-build-tools.delivery.puppetlabs.net
...
 nuget_host: nexus.delivery.puppetlabs.net:8081
-nuget_repo_path: /content/repositories/nuget-pl-build-tools
-svr4_path: /opt/build-tools/solaris
-p5p_path: /opt/build-tools/solaris
-ips_path: /export/repo-pl-build-tools
-apt_repo_path: /opt/tools/freight.build-tools/apt
-apt_repo_staging_path: /opt/tools/freight.build-tools/apt
-yum_repo_path: /opt/build-tools/yum
-ips_repo: pl-build-tools
+nuget_repo_path: /content/repositories/nuget-pl-build-tools-staging
```
Comment (melissa) [inline builder_data.yaml:12]: This is a repo on nexus, so this will have to be fixed

File: builder_data.yaml [hunk 2]
```diff
@@ -9,21 +9,21 @@ svr4_host: pl-build-tools.delivery.puppetlabs.net
...
+yum_repo_path: /opt/build-tools-staging/yum
+ips_repo: pl-build-tools-staging
 apt_repo_name: ''
 yum_repo_name: ''
 apt_repo_command: |
   keychain -k mine;
   eval $(/usr/bin/keychain -q --agents gpg --eval __GPG_KEY__);
   export GPG_TTY=$(tty);
-  sudo -E freight-cache -c /etc/freight.conf.d/pl-build-tools-repo.conf;
+  sudo -E freight-cache -c /etc/freight.conf.d/pl-build-tools-staging-repo.conf;
```
Comment (melissa) [inline builder_data.yaml:26]: What changes will have to be made to this file? Can we just keep the original?

File: builder_data.yaml (hunk 1 above)
Comment (shrug) [inline builder_data.yaml:12]: it's added to Nexus

File: builder_data.yaml (hunk 2 above)
Comment (shrug) [inline builder_data.yaml:26]: It's generated by the freight puppet module, already done

**Pull request merged**